cmd/cstor-pool-mgmt/volumereplica/volumereplica.go

```diff
@@ -107,7 +144,7 @@ func GetVolumes() ([]string, error) {
 
 // DeleteVolume deletes the specified volume.
 func DeleteVolume(fullVolName string) error {
-	deleteVolStr := []string{"destroy", "-r", fullVolName}
+	deleteVolStr := []string{"destroy", "-R", fullVolName}
```

Comment:
After discussion with @avishnu it was concluded that this change should be removed.